source/extensions/common/workload_discovery/discovery.proto

```diff
@@ -58,8 +58,7 @@ message Service {
 }
 
 message Workload {
-  reserved "uid";
-  reserved 20;
+  string uid = 20;
```

Comment:
Could you clean up before submitting by removing unused protos (service and address)

Reply:
done